OnPod/Node/Common/PeripheralInterfaces/include/I2CInterface.h

```diff
@@ -0,0 +1,53 @@
+#ifndef H_I2CINTERFACE
+#define H_I2CINTERFACE
+
+#include "main.h"
+
+class I2CInterface
+{
+    private:
+
+    // Pointer to I2C Handle Type Def structure contains the config info for the specified I2C.
+    I2C_HandleTypeDef *m_hi2c;
+    uint8_t m_dataRecieved[8]; 
+    uint8_t m_dataBufferLength = 8;
+    uint32_t m_timeout; 
```

Comment:
This value doesn't get set and is only used in the `isDeviceReady` method. I'd say your best bet is to have `isDeviceReady ` take timeout as a param like the other methods do

Reply:
I only meant to highlight line 14, this is about `m_timeout`